packages/webapp/src/util/constants.js

```diff
@@ -5,4 +5,6 @@ export const DO_ORIGIN_URL = `https://${
 export const DO_CDN_URL = `https://${
   import.meta.env.VITE_DO_BUCKET_NAME
 }.nyc3.cdn.digitaloceanspaces.com`;
-export const APP_VERSION = '3.5.1';
+
+// Changing this forces logout and updates the new release card
+export const APP_VERSION = '3.5.2.1';
```

Comment:
Will we be updating this version to 3.6 when we release the new navigation?

Reply:
Is the IA release version 3.6? I wasn't sure if it would be that or patch (3.5.3). But I guess with full release notes a minor version bump makes more sense.